src/Sarif/Writers/LogFilePersistenceOptions.cs

```diff
@@ -6,22 +6,20 @@
 namespace Microsoft.CodeAnalysis.Sarif.Writers
 {
     [Flags]
-    public enum LoggingOptions
+    public enum LogFilePersistenceOptions
```

Comment:
>LogFilePersistenceOptions [](start = 16, length = 25)

Add the rename to the releasehistory, because this will break stuff

Reply:
Done.

---
In reply to: [559825313](https://github.com/microsoft/sarif-sdk/pull/2247#discussion_r559825313) [](ancestors = 559825313)